### PR TITLE
Adding thebrain9 cask

### PR DESCRIPTION
**Pull request by enogrob**
#### Adding a new cask

- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download thebrain9` is error-free.
- [x] `brew cask style --fix thebrain9` left no offenses.
- [x] `brew cask install thebrain9` worked successfully.
- [x] `brew cask uninstall thebrain9` worked successfully.

### CODE REVIEW

File: Casks/thebrain9.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 '0b72955083b96b4f3a3f3597ee8431d5f26815b72a11b847aeb411cd089e0e94'
+
+  url 'http://api.thebrain.com/?a=doDirectDownload&id=78'
+  name 'TheBrain 9'
+  homepage 'http://www.thebrain.com/products/thebrain/thebrain9/'
+  license :commercial
+
+  app 'TheBrain 9.app'
+
+  uninstall delete: '/Applications/TheBrain 9.app'
```
Comment (reitermarkus) [inline Casks/thebrain9.rb]: This is not needed, as `uninstall` knows where the `app` was moved.

**Comment by enogrob** [comment]
You mean that I have to delete just the line 12?

**Comment by reitermarkus** [comment]
Exactly.

**Comment by reitermarkus** [comment]
Also, there seems to be a versioned URL available, so please use this: `http://updater.thebrain.com/files/TheBrain9.0.97.0.dmg`

**Comment by enogrob** [comment]
ok done!

**Pull request merged**